content/bundles_intro.tex

```diff
@@ -0,0 +1,23 @@
+\newtext{The performance of \openshmem programs that issue many consecutive and
```

Comment:
Irrespective of small/large-sized messages - bundles/sessions provides hints to the implementation to optimize. We need a better definition of what a bundle or session refers to.